.cascade-code/Chapman.edu/_components/left_nav/left_nav_children.vtl

```diff
@@ -0,0 +1,137 @@
+## Formerly LeftNavNoSiblings. Used for index pages.
+## New left nav format that works without a block
+
+#set ( $pagePath = $currentPage.path )
+#set ( $parentDir = $currentPage.parentFolder )
+
+## Builds the left navigation
+#macro ( buildLeftNav )
+    #buildLeftNavTitle
+    <ul>
```

Comment:
I'd prefer this block below were its own macro: `buildLefNavMenu` or `buildLeftNavMenu`.